reference.md

```diff
@@ -781,58 +855,3 @@ The following fields are all attributes within the main "data" object for this f
   }
 }
 ```
-
-### realtime_booking
```

Comment:
Also not strictly opposed, again just curious as to the reasoning here. What are the value adds that you see in collapsing `wait_time` and `realtime_booking`? 

Potential downside: this will lump a _lot_ of only-kinda-sorta-related data into the `wait_time` object. In turn, maybe if we do this we should rename `wait_time` to something more generic? 

Thoughts?

Reply:
estimated_wait_time, estimated_travel_time, estimated_travel_cost and estimated_travel_cost_currency are cohesive, I cannot think of a benefit to make separate http calls for this data.

The attribute booking_detail is required to support operator aggregator (ex: Montreal Taxi Registry) where estimated_wait_time and booking_detail change depending on the dispatched operator.

Equity between operators is guaranteed by always promoting the operator with the closest taxi to the customer in real time.